chat/message/src/MessageActions/MessageActions.stories.tsx

```diff
@@ -83,6 +83,145 @@ export const LiveExample: StoryObj<MessageActionsProps> = {
   },
 };
 
+export const LightModeWithCopyHover: StoryObj<MessageActionsProps> = {
+  render: Template,
+  args: {
+    onClickCopy: testOnClickCopy,
+  },
+  play: async ({ canvasElement }) => {
```

Comment:
nit: this looks like it can be turned into a reusable function that you can use for light and dark mode